### DISCUSSION TITLE
adjust the time by pressing buttons

### DISCUSSION THREAD

**Discussion by allfallsdow** [Makuna/Rtc#146] [Q&A] [answered]
Hello,first thanks for Rtc. library.Recently I'm making a program.
Q;Which funcion could I call to adjust the time second by second,or minute by minute etc?

**Comment by Makuna** [comment, marked as answer]
What do you mean by "adjust the time second by second"?  
Do you mean set the time?  Almost every example demonstrates setting the time.
```
RtcDateTime compiled = RtcDateTime(__DATE__, __TIME__); // instantiate a RtcDateTime with the new Date and Time you want (many ways)
Rtc.SetDateTime(compiled); // apply that RtcDateTime instance to the RTC module
```

If you mean, modify the existing time, then
```
RtcDateTime modify = Rtc.GetDateTime(); // get current date and time
modify += 60; // add 60 seconds
Rtc.SetDateTime(modify); // apply that modified instance to the RTC module
```
